### PR TITLE
Chore/ng17

### PR DESCRIPTION
**Pull request by jnizet**

### CODE REVIEW

File: frontend/src/styles.scss [hunk 1]
```diff
@@ -1,6 +1,6 @@
 $fa-font-path: '../fonts';
 
-@import 'font-awesome/scss/font-awesome';
+@import 'font-awesome/css/font-awesome';
```
Comment (cexbrayat) [inline frontend/src/styles.scss:3]: you should be able to use `$fa-font-path` like I did in propack (and probably need to remove styles from tests)

File: frontend/package.json [hunk 2]
```diff
@@ -27,14 +27,14 @@
...
+    "@angular/common": "17.0.3",
+    "@angular/compiler": "17.0.3",
+    "@angular/core": "17.0.3",
+    "@angular/forms": "17.0.3",
+    "@angular/localize": "17.0.3",
+    "@angular/platform-browser": "17.0.3",
+    "@angular/platform-browser-dynamic": "17.0.3",
+    "@angular/router": "17.0.3",
     "@cypress/schematic": "2.5.1",
     "@ng-bootstrap/ng-bootstrap": "15.1.2",
```
Comment (cexbrayat) [inline frontend/package.json:39]: same here: no issue with ng-bootstrap?

**Review by cexbrayat — APPROVED** [review]

File: frontend/src/styles.scss (hunk 1 above)
Comment (jnizet) [reply frontend/src/styles.scss:3]: Yes, but here the tests are more important, and I like being able to see the components in the browser when debugging them.

Comment (jnizet) [reply frontend/src/styles.scss:3]: Oh yes, indeed.

**Pull request merged**